truss/constants.py

```diff
@@ -34,6 +34,9 @@
 SUPPORTED_PYTHON_VERSIONS = {"3.8", "3.9", "3.10", "3.11"}
 
 TRTLLM_PREDICT_CONCURRENCY = 512
+TRTLLM_MIN_MEMORY_REQUEST_GI = 24
```

Comment:
@bdubayah @joostinyi In July 2024 we introduced min 24GB. Nowadays, we have L4:1, which makes a nice deployment for 1B fp8 models. Lets lower that to 16GB?